jmaqs-webservices-jdk11/pom.xml

```diff
@@ -12,6 +12,14 @@
     <groupId>com.magenic.jmaqs.webservices</groupId>
     <artifactId>jmaqs-webservices-jdk11</artifactId>
     <version>2.0.1-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>com.magenic.jmaqs.utilities</groupId>
+            <artifactId>jmaqs-utilities</artifactId>
+            <version>2.0.1-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```

Comment:
Put this below the build block.  It looks really gross in the middle of the module definition information.